docs-3.x/guides/setup-lando-on-windows-with-wsl-2.md

```diff
@@ -70,6 +70,44 @@ You'll want to do all of your shell activity from within the WSL system, and you
 ## Some caveats
 
 Performance is generally very good with this approach, however, stability can at times leave something to be be desired. Occassional freezes under heavy load are fairly commonplace, and seem to be resolved by rebooting the system. All things considered, the reduction in CPU load, battery drain, and the accompanied increase in the speed of every single action you take (accessing site pages, running any CLI commands) more than outweighs the disruption of these occasional issues for most users. Hopefully stability will continue to increase with time.
+            
+## Configure host IP
+
+If you are running Lando inside WSL you have to override the `LANDO_HOST_IP` to use Xdebug because by default Lando uses the WSL IP which can switch between reboots of WSL.
+Also, you have to differentiate between running your IDE (e.g. PHPStorm) in Windows or inside WSL.
+
+### 1. Add this lines to `~/.bashrc` inside WSL2 :
+a) For IDE in Windows:
+```bash
+# Set correct dev host IP to Windows
+export LANDO_HOST_IP_DEV=host.wsl.internal
+sudo sed -i "/$LANDO_HOST_IP_DEV/d" /etc/hosts && sudo sh -c "echo $(cat /etc/resolv.conf | grep nameserver | awk '{print $2; exit;}') $LANDO_HOST_IP_DEV >> /etc/hosts"
```

Comment:
Today we realized that this seems to be a bad solution. Maybe only the following is the solution and you always need to to a `lando rebuild` if the IP of wsl or of the wsl host has changed 😢 
```suggestion
export LANDO_HOST_IP_DEV=$(cat /etc/resolv.conf | grep nameserver | awk '{print $2; exit;}')
```